### DISCUSSION TITLE
modify the "variants" property of the Navbar to "floating"

### DISCUSSION THREAD

**Discussion by peterchui** [heroui-inc/heroui#1192] [Help]
Hello everyone,

When I modify the "variants" property of the Navbar to "floating" in my code, the Navbar scrolls along with the page. The same thing happens with "sticky". I can't figure out the exact reason for this. Has anyone else had a similar experience? I would appreciate any help.

Thank you.

**Comment by LifeTakerSimo** [comment]
Hi, i'm using floating variant but the navbar does not follow along.
